Sprintfinity3902/Interfaces/IDungeon.cs

```diff
@@ -1,19 +1,19 @@
 ﻿using Microsoft.Xna.Framework;
 using Microsoft.Xna.Framework.Graphics;
-using Sprintfinity3902.Sprites;
-using System;
-using System.Collections.Generic;
-using System.Text;
 
 namespace Sprintfinity3902.Interfaces
 {
     public interface IDungeon
     {
+        public IRoom CurrentRoom { get; set; }
```

Comment:
I think the method below can help to keep this variable private to the Dungeon entities.